readme.md

```diff
@@ -36,40 +36,70 @@ emitter.emit('🦄', '🌈');
 
 The above only works in Node.js 8 or newer. For older Node.js versions you can use `require('emittery/legacy')`.
 
-
 ## API
 
 ### emitter = new Emittery()
 
-#### on(eventName, listener)
+#### on([eventName], listener)
```

Comment:
The `[ ]` notation is more commonly used to indicate optional parameter. You need to find a better way to document this.